easybuild/easyconfigs/v/vcflib/vcflib-1.0.2-GCC-10.2.0.eb

```diff
@@ -49,9 +49,13 @@ preconfigopts += " -e 's|multichoose.h|multichoose/multichoose.h|g' -e 's|fileve
 preconfigopts += " -e 's|tabix.hpp|tabixpp/tabix.hpp|g' -e 's|Fasta.h|fastahack/Fasta.h|g'"
 preconfigopts += " -e 's|disorder.h|smithwaterman/disorder.h|g' {} \; && "
 
+postinstallcmds = ["cp -r %(builddir)s/%(name)s-%(version)s/scripts %(installdir)s"]
+
+modextrapaths = {'PATH': ['scripts']}
+
 sanity_check_paths = {
     'files': ['bin/vcfannotate', 'bin/vcfcombine', 'lib/libvcflib.%s' % SHLIB_EXT],
-    'dirs': [],
+    'dirs': ['bin', 'lib', 'scripts'],
```

Comment:
Just `scripts` is sufficient here, since we're already checking for specific files in `bin` and `lib`